packages/builder/src/components/ContextMenu.svelte

```diff
@@ -0,0 +1,68 @@
+<script>
+  import { contextMenuStore } from "stores/builder"
+  import { Popover, Menu, MenuItem } from "@budibase/bbui"
+
+  let dropdown
+  let anchor
+
+  const showDropdown = dropdown => {
+    dropdown?.show()
+  }
+
+  $: showDropdown(dropdown)
+
+  const handleKeyDown = () => {
+    if ($contextMenuStore.visible) {
+      contextMenuStore.close()
+    }
+  }
+
+  const handleItemClick = async itemCallback => {
+    await itemCallback()
+    contextMenuStore.close()
+  }
+</script>
+
+<svelte:window on:keydown={handleKeyDown} />
+<div
+  bind:this={anchor}
+  class="anchor"
+  style:top={`${$contextMenuStore.position.y}px`}
+  style:left={`${$contextMenuStore.position.x}px`}
+/>
+
+{#key $contextMenuStore.position}
+  {#if $contextMenuStore.visible}
+    <Popover
+      bind:this={dropdown}
+      {anchor}
+      resizable={false}
+      align="left"
+      on:close={contextMenuStore.close}
```

Comment:
Maybe just personal preference, but I think this is much nicer if we remove the animation with `animate={false}`. It's just much snappier, and feels more typical for a context menu.

Reply:
Don't disagree, though I think no animation at all looks a little weird too. I changed it be a bit more like the MacOS right-click menu animation, where it's a subtle fade in, bit longer on the out.

You can let me know if you think that looks better, if not I'll just remove the animation entirely.